src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -43,10 +43,6 @@
          * to think about number formatting issues.
          */
         Utils.forceJVMLocaleToUSEnglish();
-
-        // Turn off the Picard legacy parser and opt in to Barclay syntax for Picard tools. This should be replaced
-        // with a config setting once PR https://github.com/broadinstitute/gatk/pull/3447 is merged.
-        System.setProperty("picard.useLegacyParser", "false");
```

Comment:
Do we actually want to allow users to run Picard tools via GATK using the legacy syntax @cmnbroad ?